nestkernel/recordables_map.h

```diff
@@ -107,7 +103,7 @@ class RecordablesMap : public std::map< Name, double ( HostNode::* )() const >
 private:
   //! Insertion functions to be used in create(), adds entry to map and list
   void
-  insert_( const Name& n, const DataAccessFct f )
+  insert_( const std::string& n, const DataAccessFct f )
```

Comment:
In l. 112 there is a `LiteralDatum` and in l. 122 an `ArrayDatum` remaining.

Reply:
Thanks for catching! Replaced now with standard data types in the comments there; a reminder to improve code there in a future step.